### PR TITLE
shortcuts

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: engine/src/factions.ts [hunk 1]
```diff
@@ -1,61 +1,75 @@
 import { Faction, Planet } from "./enums";
 
-const factions: { [key in Faction]: { name: string; planet: Planet } } = {
+const factions: { [key in Faction]: { name: string; planet: Planet; shortcut: string } } = {
   [Faction.Terrans]: {
     name: "Terrans",
     planet: Planet.Terra,
+    shortcut: "t",
```
Comment (coyotte508) [inline engine/src/factions.ts:7]: I think that shortcuts should rather be in the viewer

**Comment by coyotte508** [comment]
This is a really nice feature (keyboard navigation!)

The shortcuts should be documented somewhere, at least their existence. Maybe in a tooltip/`title` over the buttons, maybe there should be a message saying that shortcuts are available, maybe there should be a key/trigger that displays all shortcuts currently usable

**Comment by zeitlinger** [comment]
> This is a really nice feature (keyboard navigation!)
> 
> The shortcuts should be documented somewhere, at least their existence. Maybe in a tooltip/`title` over the buttons, maybe there should be a message saying that shortcuts are available, maybe there should be a key/trigger that displays all shortcuts currently usable

I hope I found a way. Inspired by

image

**Comment by coyotte508** [comment]
> I hope I found a way. Inspired by
> 
> image

This is actually really neat!

**Pull request merged**